doc/src/site/sphinx/outputs.rst

```diff
@@ -323,6 +323,9 @@ The parquet output uses generic implementation of DataFrames.
 | path                     | Destination path to store info.               | No       |                       |
 +--------------------------+-----------------------------------------------+----------+-----------------------+
 
+In Parquet each cube is storage in one path, but when you modify the policy that involve this cube, Sparkta create a
```

Comment:
When you are using Parquet as output, it will save the result in a path concatenating a base path, the dimension names of the cube and a version starting in one.

Example: if the basePath is /user/stratio, the cube has 2 dimensions (name, price) and it is the first time that you save the policy then the final path will be /user/stratio/name_price_v1
If  you modify some dimension of this cube then the version will be increased in one: /user/stratio/name_price_v2